denoise/driver copy.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
```

Comment:
Header block should have more info. Block would include authorship, license, copyright, dates and description of what the driver does.